contracts/lib/Ownable.sol

```diff
@@ -12,8 +12,12 @@ contract Ownable {
     _;
   }
 
-  function transferOwnership(address _newOwner) onlyOwner {
-    require(_newOwner != address(0));
+  modifier validAddress(address _address) {
+    require(_address != address(0));
+    _;
+  }
```

Comment:
I'm fine with this for now, but it's a bit weird for this modifier to be here.

Reply:
Yeah I agree, wasn't sure where to put it.  Think we need a separate file for this type of stuff?  Thought about `AddressUtils` but didn't work from there.

Reply:
Happy to make the appropriate change if you want to do so now.

Reply:
Wondering if there is a publicly available modifier like this.